src/mixer.erl

```diff
@@ -99,11 +102,26 @@ group_mixins({_CMod, _}, [#mixin{mod=Mod, line=Line}=H|T], Accum) ->
 expand_mixin(Line, Name) when is_atom(Name) ->
     case catch Name:module_info(exports) of
```

Comment:
Remove the dynamic function call on line 103. Only modules that define callbacks should make dynamic calls.